web-ui/src/main/resources/ui/app/Visualization.js

```diff
@@ -44,8 +44,6 @@
  */
  define(function(require) {
```

Comment:
Well, it actually has a point - we don't use require in this function. I'll fix it later today.